### PR TITLE
#158428621 Passenger can make requests on driver's ride offer and view ride history

### PR DESCRIPTION
**Pull request by rethes**
1. What does this PR do?
Create a page that displays passenger ride history
Create a page that displays all requested rides

2. Description of Task to be completed?
Passengers can view their ride history
Passengers can view the requested rides
Passengers can make a request on a ride offer

3. What are the relevant pivotal tracker stories?
#158428621 Passenger can make requests on driver's ride offer and view ride history

5. Screenshots 
Passengers can view their ride history
image

Passengers can view the requested rides
image

Passengers can make a request on a ride offer 
image

### CODE REVIEW

File: Designs/UI/passengers/ridehistory.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>Ride History Passenger | Ride-my-way </title>
+    <link rel="stylesheet" href="../assets/css/style.css">
+    <style>.item2 { grid-area: menu; }
```
Comment (JoshuaOndieki) [inline Designs/UI/passengers/ridehistory.html]: Separate your styling from the HTML in a different file.

**Comment by xcixor** [comment]
Great progress but if I may suggest would you have a link for all the available menus in the navbar so that its easy for anyone to check them out?

File: Designs/UI/passengers/ridehistory.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  text-align: center;
+height: auto;
+  padding: 0px 0;
+  font-size: 30px;
+}
+</style>
+</head>
+<body>
+<div class="nav " style="color: #f2f2f2">
+    <a href="../../../index.html" style="float:left ;color: #f2f2f2;   background-color:#e01f3d; ">Ride My Way</a>
```
Comment (JoshuaOndieki) [inline Designs/UI/passengers/ridehistory.html]: Avoid inline css

**Comment by JoshuaOndieki** [comment]
In your ride details, be more specific on the date field. Is it the date when the ride was posted or when it will take place?

**Comment by Xerrex** [comment]
Maybe to help out with the mardown checkout [this cheat sheet](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf)

**Pull request merged**